Fix for collation errors when importing catallog tables

rafatower opened the pull request:
This fixes #393 by avoiding collation options. The affected function
is only used from diagnostics, and aside from `pg_extension` only
meant to import other catallog tables for internal
diagnostics (`pg_catalog` and `pg_settings`) so it shall be pretty
safe.

Commit:
Fix for collation errors when importing catallog tables

This fixes #393 by avoiding collation options. The affected function
is only used from diagnostics, and aside from `pg_extension` only
meant to import other catallog tables for internal
diagnostics (`pg_catalog` and `pg_settings`) so it shall be pretty
safe.

Algunenano commented:
Can you explain why this is needed here and not in the other places where `IMPORT FOREIGN SCHEMA` is called?

rafatower commented:
Because of default collations, checks in FDW's, and catallog tables using `NAME` type.

Algunenano commented:
I'm sorry but I don't understand it. What about a foreign table that uses NAME as one of the column, is that different? How?

rafatower commented:
I suspect it is not different at all, just less common

Commit:
Apply a similar patch to avoid collation issues

With this, Josema could keep testing, listing remote tables without
otherwise ERROR in DB and 500 error in RoR.
